addons/website_event_track/models/event_track.py

```diff
@@ -43,9 +43,9 @@ def _get_default_stage_id(self):
     user_id = fields.Many2one('res.users', 'Responsible', tracking=True, default=lambda self: self.env.user)
     company_id = fields.Many2one('res.company', related='event_id.company_id')
     partner_id = fields.Many2one('res.partner', 'Speaker')
-    partner_name = fields.Char('Name')
-    partner_email = fields.Char('Email')
-    partner_phone = fields.Char('Phone')
+    partner_name = fields.Char('Name', tracking=True)
```

Comment:
You could set an integer to order tracking values. That way it is so much nice.

Reply:
Oooooh, I didn't knooooow. Waaaaaouw